### ISSUE TITLE
gyro braking is broken

### ISSUE THREAD

**Issue by deltanedas** [space-wizards/space-station-14#16762]
## Description
its pretty much press space to randomize your velocity
its very rare that it will actually bring you to a stop instead of making you spin randomly

**Reproduction**
1. make basic shittle as shown
2. spin
3. try to brake

![04:07:18](https://github.com/space-wizards/space-station-14/assets/39013340/3f4c5b9a-8866-454d-beb6-7bcc71c855c5)

**Comment by metalgearsloth** [comment]
shittle issue.

I'll take a look.

**Comment by metalgearsloth** [comment]
Can't repro.

**Comment by metalgearsloth** [comment]
Does it happen with https://github.com/space-wizards/space-station-14/pull/16774

**Comment by deltanedas** [comment]
ill see

**Comment by deltanedas** [comment]
@metalgearsloth yep it is fixed now, braking sets angular velocity to 0 as expected

**Comment by metalgearsloth** [comment]
sounds like merge approval to me
